interfaces/mount/entry.go

```diff
@@ -103,3 +108,36 @@ func (e Entry) String() string {
 	return fmt.Sprintf("%s %s %s %s %d %d",
 		name, dir, fsType, options, e.DumpFrequency, e.CheckPassNumber)
 }
+
+// ParseEntry parses a fstab-like entry.
+func ParseEntry(s string) (Entry, error) {
+	var e Entry
+	var err error
+	var df, cpn int
+	fields := strings.FieldsFunc(s, func(r rune) bool { return r == ' ' || r == '\t' })
+	// do all error checks before any assignments to `e'
+	if len(fields) < 4 || len(fields) > 6 {
+		return e, fmt.Errorf("expected between 4 and 6 fields, found %d", len(fields))
+	}
+	// Parse DumpFrequency if we have at least 5 fields
+	if len(fields) >= 5 {
```

Comment:
In general `len(fields) > N` when you want to use N is easier to read than >= and N-1. So `len(fields) > 5` allows you to use `fields[5]` for example.